app/data/playfair/playfairImportExport.json

```diff
@@ -0,0 +1,232 @@
+[
```

Comment:
If we do a refactoring pass in the future, I suspect part of it will be turning this file from an array of objects to an object of arrays, i.e. an object like:

```
{ "years": [1700, 1710, 1720, ...], "imports": [300000, 310000, 500000, ...] }
```

I'm suggesting this because it appears that there's a lot of mapping over this array and getting the imports as an array.